src/lib/components/Button/Button.scss

```diff
@@ -1,24 +1,9 @@
 @use "sass:map";
 
-// TODO: replace with global color variables from src/variables.scss
+// TODO: Bring the below colors inline with global colors in `src/variables.scss`
 $colors: (
-  "primary": #0051ad,
-  "primary-dark": #063c7a,
-  "primary-darker": #00284d,
-  "secondary": #ffc745,
-  "secondary-dark": #ffb200,
-  "secondary-darker": #d49400,
-  "disabled": #c9c9c9,
-  "disabled-dark": #adadad,
-  "base-lightest": #f2f1f0,
-  "base-light": #a9aeb1,
-  "base-dark": #565c65,
-  "base-darker": #3d4551,
-  "gray-05": #f0f0f0,
-  "primary-lightest": #d1e9ff,
   "text-only-hover": rgba(6, 60, 122, 16%),
   "text-only-active": rgba(6, 60, 122, 30%),
-  "outline-inverse": #dcdee0,
```

Comment:
`outline-inverse` was removed since it wasn't being used after we corrected some things to use `base-light` instead.